pyaedt/application/MessageManager.py

```diff
@@ -309,7 +309,10 @@ def add_message(self, type, message_text, level=None, proj_name=None, des_name=N
                 des_name = self._design_name
             if des_name and ";" in des_name:
                 des_name = des_name[des_name.find(";") + 1 :]
-            self._desktop.AddMessage(proj_name, des_name, type, message_text)
+            try:
+                self._desktop.AddMessage(proj_name, des_name, type, message_text)
+            except:
+                print("pyaedt info: Failed in Adding Desktop Message")
```

Comment:
Is there a way we can raise exceptions instead of printing?

@MaxJPRey, can you make sure these get either logged or raised?

Reply:
@akaszynski Sure Alex. I will.

Reply:
Actually that is the MessageManager itself that we are wrapping with the standard python logger.

Reply:
Exactly. We don't have print anywhere else.